server/utils/middleware/ratingMiddleware.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @object
+ */
+const ratingMiddleware = {
+  /**
+   * @param {*} req the request body
+   * @param {*} res the res body
+   * @param {*} next a call to the next middle function
+   */
+  validateRating(req, res, next) {
+    const numberFormat = /^\d*[1-9]\d*$/;
+    if (
+      (numberFormat.test(req.params.slug))
```

Comment:
Why are you testing the slug?

Reply:
I am testing to prevent digit being entered as the slug. i.e 1, 4 etc.